platform/ethereum/opensea/client.go

```diff
@@ -20,10 +26,10 @@ func (c Client) GetCollections(owner string) (page []Collection, err error) {
 	return
 }
 
-func (c Client) GetCollectibles(owner string, collectibleID string) ([]Collectible, error) {
+func (c Client) getCollectibles(owner string, collectionID string) ([]Collectible, error) {
```

Comment:
since this is opensea client it probably make sense to have those methods public

Reply:
done